examples/magento-graphcms/components/Layout/LayoutFull.tsx

```diff
@@ -63,6 +64,11 @@ export function LayoutFull(props: LayoutFullProps) {
                 <IconSvg src={iconCustomerService} size='large' />
               </Fab>
             </PageLink>
+            <PageLink href='/wishlist' passHref>
+              <Fab aria-label={t`Wishlist`} size='large' color='inherit'>
+                <IconSvg src={iconHeart} size='large' />
+              </Fab>
+            </PageLink>
```

Comment:
Implement a badge: If you take a look at the <CustomerFab /> you'll see an implementation how it has been handled here to add a DesktopHeaderBadge (this one is style differently from a regular badge)